application/common/core.cpp

```diff
@@ -43,19 +44,61 @@ void loop()
 
     // Get the stage as it will receive most events.
     Loom2D::Stage *stage = Loom2D::Stage::smMainStage;
-    
+
+    if(gPendingResizeEvent && stage)
```

Comment:
Thank you. However. You were TOTALLY right and I did have to do it elsewhere in the stage logic. Fixed now.